Documentation/concepts/networking/ipam/kubernetes.rst

```diff
@@ -37,12 +37,16 @@ Field                Description
 
 **via v1.Node annotation**
 
-=================================== ============================================================
-Annotation                          Description
-=================================== ============================================================
-``io.cilium.network.ipv4-pod-cidr`` IPv4 PodCIDR range
-``io.cilium.network.ipv6-pod-cidr`` IPv6 PodCIDR range
-=================================== ============================================================
+====================================== ==========================================================
+Annotation                             Description
+====================================== ==========================================================
+``io.cilium.network.ipv4-pod-cidr``    IPv4 PodCIDR range
+``io.cilium.network.ipv6-pod-cidr``    IPv6 PodCIDR range
+``io.cilium.network.ipv4-cilium-host`` IPv4 address of the cilium host interface
+``io.cilium.network.ipv6-cilium-host`` IPv6 address of the cilium host interface
+``io.cilium.network.ipv4-health-ip``   IPv4 address of the cilium-health endpoint
+``io.cilium.network.ipv6-health-ip``   IPv6 address of the cilium-health endpoint
+=====================================  ==========================================================
```

Comment:
Maybe this would fix the error:
```suggestion
====================================== ==========================================================
```

Reply:
Thanks, yes it does actually!